api/services/AccessControlService/EventAccessControl/allowClientToEditEvent.ts

```diff
@@ -1,4 +1,4 @@
-import { addUserRoles } from './operations';
+import { addUserRoles } from '../operations';
```

Comment:
When the imported files are in the same directory, you can use `from './operations'` for simplicity. When they are in different directories, however, use `import { addUserRoles } from '@Services/AccessControlService/operations'` instead.